mlir/lib/Target/SPIRV/Deserialization/Deserializer.cpp

```diff
@@ -2158,13 +2158,50 @@ LogicalResult spirv::Deserializer::wireUpBlockArgument() {
   return success();
 }
 
+LogicalResult spirv::Deserializer::splitConditionalBlocks() {
+  for (auto it = blockMergeInfo.begin(); it != blockMergeInfo.end(); it++) {
```

Comment:
```suggestion
  for (auto it = blockMergeInfo.begin(); it != blockMergeInfo.end(); ++it) {
```
See https://llvm.org/docs/CodingStandards.html#prefer-preincrement